backend/utils/sidebar.py

```diff
@@ -0,0 +1,183 @@
+from utils.room_utils import DB, DEFAULT_DM_IMG, get_org_rooms
+
+
+class Sidebar:
+    """Serves as a data source for the sidebar
+
+    Sidebar class helps makes faster connection to core,
+    sorts out data, and creates the sidebar data format
+    for the frontend
+    """
+
+    @classmethod
+    async def __get_room_members(
+        cls, member_id: str, room: dict, org_members: list
```

Comment:
The room is needed to be in a dict type, because it shouldn't be validated as an object of Room, as the data the function will be receiving will be in a dict.

org_members is required in order to perform logic of getting all room members details for the sidebar, such as username, and profile picture to be rendered